### ISSUE TITLE
Difference between the name on pypi and the name of the module

### ISSUE THREAD

**Issue by luzfcb** [jrief/django-angular#97]
Although I have followed "everything" as the documentation, nothing worked properly. 

After racking my brain for some time, I discovered my mistake. 
I had installed the wrong package, they just ran `pip install djangular`, and a package[1] created by another developer was installed.

I think the module of this package Should be renamed to django_angular. This would avoid confusion stupid like that, and to facilitate that IDE's like PyCharm detect and install the correct package. 

In short, I believe the pypi package name and the module name must be the same.

[1] https://pypi.python.org/pypi/djangular/0.2.7
[2] https://pypi.python.org/pypi/django-angular/0.7.6

**Comment by adrienbrunet** [comment]
Module name will be updated with the new release.

**Comment by adrienbrunet** [comment]
Cf ticket #239
